miden-tx/src/tests/kernel_tests/mod.rs

```diff
@@ -28,6 +28,10 @@ mod test_tx;
 // HELPER FUNCTIONS
 // ================================================================================================
 
+pub fn read_root_mem_value<H: Host>(process: &Process<H>, addr: u32) -> Word {
+    process.get_mem_value(ContextId::root(), addr).unwrap()
+}
```

Comment:
We have the same function [here](https://github.com/0xPolygonMiden/miden-base/blob/12760b0dca6aab33aa14ef6122bfd53d922470da/miden-tx/src/tests/kernel_tests/test_tx.rs#L558). We should probably keep just one copy.